modules/react/popup/lib/Popper.tsx

```diff
@@ -154,10 +154,8 @@ const OpenPopper = React.forwardRef<HTMLDivElement, PopperProps>(
         enabled: true,
         phase: 'afterWrite',
         fn({state}) {
-          if (placementRef.current !== state.placement) {
-            setPlacement(state.placement);
-            onPlacementChange?.(state.placement);
-          }
+          setPlacement(state.placement);
```

Comment:
add if statement back in

Reply:
The if statement didn't allow the functions to run to reset the popup. So it will stay removed.